src/Symfony/Component/HttpClient/Tests/TraceableHttpClientTest.php

```diff
@@ -80,4 +82,20 @@ public function testItResetsTraces()
         $sut->reset();
         $this->assertCount(0, $sut->getTracedRequests());
     }
+
+    /**
+     * @requires extension curl
```

Comment:
would work with NativeHttpClient, allowing to remove this?

Reply:
It seems not, tried with NativeHttpClient before but the response isn't fully chunked (`['', 'Symfony is awesome!', '']`);

Reply:
But I can use this partially chunked response with NativeHttpClient if you prefer :)

Reply:
or we can compare after imploding, don't you think?

Reply:
Yes we can do that but this doesn't highlight chunked behavior, does it ?

Reply:
we could also assert that the size of the array is > 1 then
but where exactly chunks are split is behavioral, it cannot be tested eactly since there are no contracts about this.

Reply:
So let's go with that :)